src/Gateway.php

```diff
@@ -15,6 +15,15 @@ public function __construct ( $host, $request_options = array())
         $this->description = 'Endpoint SDK for using the gateway';
     }
 
+    public function login($credentials) {
+        if (!is_array($credentials))
```

Comment:
I know the syntax is taken from existing functions in this package but I strongly prefer not excluding brackets

Reply:
Agree, will change for this PR.